src/main-mw.ts

```diff
@@ -18,7 +19,13 @@ export default function(): Middleware {
     }),
     problem(),
     session({
-      store: 'memory',
+      store: process.env.REDIS_SESSION ? new RedisStore({
+        prefix: 'A12N-session',
+        clientOptions: {
+          port: process.env.REDIS_PORT ? parseInt(process.env.REDIS_PORT, 10) : 6379,
+          host: process.env.REDIS_HOST ? process.env.REDIS_HOST : '127.0.0.1',
+        },
+      }) : 'memory',
```

Comment:
Can you remove the `REDIS_SESSION` environment variable, and just use the existence of `REDIS_HOST` as a way to determine we're going to use Redis?

Reply:
yep!